### PR TITLE
lesson8.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Instructions:
Write a program that initializes a string with the value of "Eat Work Play Sleep repeat". Then, use string manipulation commands to get the string to read as: "working playing".

### CODE REVIEW

File: lesson8.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@author: amyhaddad
+"""
+
+a = "Eat Work Play Sleep repeat"
+
+print("Here's option 1:")
+index_work = a[4:8:1] + "ing"
+index_play = a[9:13:1] + "ing"
+
+print(index_work.lower(), index_play.lower())
```
Comment (paulghaddad) [inline lesson8.py]: Nice use of `lower()`! 👏

File: lesson8.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+print("Here's option 2:")
+working_playing = a.replace("Eat Work Play Sleep repeat", "working playing")
+print(working_playing)
+
+print("Here's option 3:")
```
Comment (paulghaddad) [inline lesson8.py]: Really nice job! 👏

File: lesson8.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+a1 = a.replace("Eat ", "")
```
Comment (paulghaddad) [inline lesson8.py]: Consider a better named variable.

File: lesson8.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+a = "Eat Work Play Sleep repeat"
+
+print("Here's option 1:")
+index_work = a[4:8:1] + "ing"
+index_play = a[9:13:1] + "ing"
+
+print(index_work.lower(), index_play.lower())
+
+print("Here's option 2:")
```
Comment (paulghaddad) [inline lesson8.py]: Options 1 and 3 are the best solutions for the problem.

**Comment by amymhaddad** [comment]
@paulghaddad I updated this exercise.

File: lesson8.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+test_string = "Eat Work Play Sleep repeat"
+
+# Option 1: using index
+first_word_test_string = test_string[4:8]+"ing"
```
Comment (paulghaddad) [inline lesson8.py]: Make sure you have spaces between `test_string[4:8]` and `"ing"`. This comment applies for line 13; line 14 is good!

File: lesson8.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+second_word_test_string = test_string[9:13]+"ing"
+total_word_test_string = first_word_test_string.lower() + ' ' + second_word_test_string.lower()
+print(total_word_test_string)
+
+# Option 2: spliting the string and indexing it
+new_phrase = test_string.split(' ')
+word_one_new_phrase = new_phrase[1]+"ing"
```
Comment (paulghaddad) [inline lesson8.py]: Spacing

File: lesson8.py [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+word_two_new_phrase = new_phrase[2]+"ing"
+both_words_new_phrase = word_one_new_phrase.lower() + ' ' + word_two_new_phrase.lower()
+print(both_words_new_phrase)
+
+# Option 3: using replace
```
Comment (paulghaddad) [inline lesson8.py]: Nice job using `replace`!

File: lesson8.py [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+test_string = "Eat Work Play Sleep repeat"
+
+# Option 1: using index
+first_word_test_string = test_string[4:8]+"ing"
+second_word_test_string = test_string[9:13]+"ing"
+total_word_test_string = first_word_test_string.lower() + ' ' + second_word_test_string.lower()
+print(total_word_test_string)
+
+# Option 2: spliting the string and indexing it
+new_phrase = test_string.split(' ')
```
Comment (paulghaddad) [inline lesson8.py]: Excellent!

**Comment by amymhaddad** [comment]
@paulghaddad I updated the spacing on this exercise.

**Pull request merged**